src/unity/lib/visualization/plot/plot.hpp

```diff
@@ -0,0 +1,42 @@
+#ifndef __TC_PLOT_HPP_
+#define __TC_PLOT_HPP_
+
+#include <unity/lib/toolkit_class_macros.hpp>
+#include <unity/lib/toolkit_function_macros.hpp>
+#include <unity/lib/visualization/transformation.hpp>
+#include <string>
+#include <export.hpp>
+
+namespace turi {
+  namespace visualization {
+
+    class EXPORT Plot: public toolkit_class_base{
+      public:
+
+        Plot(){};
+
+        Plot(const std::string path_to_client, const std::string vega_spec, std::shared_ptr<transformation_base> transformer, double size_array):
```

Comment:
For readability, please put each argument on its own line.

Reply:
And you should probably take a const reference to std::strings, not just a const string.